classes/classes/Items/Mutations.as

```diff
@@ -9212,10 +9454,17 @@ public function wolfPepper(type: Number, player: Player): void {
 					player.wingType = WING_TYPE_BAT_LIKE_TINY;
 					outputText("bat-like demon-wings!", false);
 				}
-				else if (player.wingType == WING_TYPE_BEE_LIKE_SMALL || player.wingType == WING_TYPE_BEE_LIKE_LARGE) {
+				//No wings
+				else if (player.wingType == WING_TYPE_NONE) {
+					outputText("\n\n", false);
+					outputText("A knot of pain forms in your shoulders as they tense up.  With a surprising force, a pair of small demonic wings sprout from your back, ripping a pair of holes in the back of your " + player.armorName + ".  <b>You now have tiny demonic wings</b>.", false);
+					player.wingType = WING_TYPE_BAT_LIKE_TINY;
+				}
+				//Other wing types
+				else {
 					outputText("\n\n", false);
 					outputText("The muscles around your shoulders bunch up uncomfortably, changing to support your wings as you feel their weight increasing.  You twist your head as far as you can for a look and realize they've changed into ", false);
-					if (player.wingType == WING_TYPE_BEE_LIKE_SMALL) {
+					if (player.wingType == (WING_TYPE_BEE_LIKE_SMALL || WING_TYPE_HARPY || WING_TYPE_DRACONIC_SMALL || WING_TYPE_IMP)) {
```

Comment:
Replace this with
```as3
					if ([WING_TYPE_BEE_LIKE_SMALL, WING_TYPE_HARPY, WING_TYPE_DRACONIC_SMALL, WING_TYPE_IMP].indexOf(player.wingType) != -1) {
```
PS: WING_TYPE_HARPY is for monsters only, so you could probably omit them here.
A couple weeks ago I've fixed every custom character that gained WING_TYPE_HARPY to gain WING_TYPE_FEATHERED_LARGE instead.

I'll probably add some code later, that replaces WING_TYPE_HARPY with WING_TYPE_FEATHERED_LARGE upon loading an old savegame.